src/index.html

```diff
@@ -7,12 +8,67 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
```

Comment:
The stylesheet link references a SCSS file (`index.scss`). Browsers cannot directly interpret SCSS files; they require CSS. Ensure that the SCSS file is compiled into a CSS file and update the `href` attribute to point to the resulting CSS file.